### PR TITLE
Refine azure-private-dns tutorial

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: docs/tutorials/azure-private-dns.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Finally, create the mentioned link with the VNET.
 
 ```
 $ az network private-dns link vnet create -g externaldns -n mylink \
-   -z example.com -v myvnet
+   -z example.com -v myvnet --registration-enabled false
```
Comment (timja) [inline docs/tutorials/azure-private-dns.md:57]: Required on my version of the azure cli, tested on 2.0.72 and 2.0.76

File: docs/tutorials/azure-private-dns.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ More powerful role-assignments like `owner` or assignments on subscription-level
 
 Start off by **creating the service principal** without role-assignments.
 ```
-$ az ad sp create-for-rbac -n externaldns-sp
+$ az ad sp create-for-rbac --skip-assignment -n http://externaldns-sp
```
Comment (timja) [inline docs/tutorials/azure-private-dns.md:69]: the docs above say skip assignment, but the cli call was missing it so it was assigning contributor at the subscription and without the http there was a warning saying it was invalid, and was being automatically fixed for you

File: docs/tutorials/azure-private-dns.md [hunk 3]
```diff
@@ -82,17 +82,11 @@ But first **retrieve the ID's** of the objects to assign roles on.
 
 ```
 # find out the resource ids of the resource group where the dns zone is deployed, and the dns zone itself
-$ az group show --name externaldns
-{
-  "id": "/subscriptions/id/resourceGroups/externaldns",
-  ...
-}
+$ az group show --name externaldns --query id -o tsv
```
Comment (timja) [inline docs/tutorials/azure-private-dns.md:85]: cleaner

File: docs/tutorials/azure-private-dns.md (hunk 2 above)
Comment (saidst) [reply docs/tutorials/azure-private-dns.md:69]: Thx!
You went through it very carefully.

File: docs/tutorials/azure-private-dns.md (hunk 3 above)
Comment (saidst) [reply docs/tutorials/azure-private-dns.md:85]: indeed.

**Comment by saidst** [comment]
thanks for reviewing.
i accept the changes.

**Pull request merged**